Add Beatmap Spotlights: Autumn 2019 news post (#2984)
ppy/osu-wiki: news/2020-01-05-beatmap-spotlights-autumn-2019.md

To mention the other gimmick / interesting thing the map has, it is impossible not to mention the slider's velocity changes. as you can see in editor, you will see a lot of green line (inherent timing points) and most of them are mainly for slider velocity's changes!  By this I mean that the mapper tried a lot of Slider Velocity combinations through numerous trial and errors to find the optimal ones, which takes a lot of effort and cute too! Especially at the build up part before alternator style section, you will see that the speed changes are fluctuating really frequently that sometimes it drops from 4.00x to 1.50x. And they are very nice and interesting decision in terms of that these pattern try to capture literally every features from the song. For example, the map uses really high SV when factors such as guitar, snare sound, and etc are mixed whereas when the beat has monotonous participants it only provides slow SV which is really representing the song well! To consider all of those, there's absolutely no way that this map will fail to amaze you.

> _written by [/_Epreus](https://osu.ppy.sh/users/7342798)_

Reviewer: ```suggestion
_written by [\_Epreus](https://osu.ppy.sh/users/7342798)_
```

escape using the backslash `\`

Author: Fudge, I have a bunch of these

Each difficulty contains excellent usage of slider velocity changes to give the player more depth in relation to both the map and music. Each difficulty is beautifully crafted and shows deep understanding of what proper structure in a map should be. Something this set does which not many people have done this year is make their lower difficulties enjoyable, and it's clear that all the mappers involved in this mapset have spent a lot of time and care making sure every difficulty is to the highest of standards. Breaks in every difficulty are utilised well and are executed in a way which leaves no room for boredom throughout the entire map. 

The top diff, Synthesia, has players having to hit many different snappings and watching Genjuro's favourite weapon of choice: Slider Velocity, which gives the player an immersive experience and allows them to become indulged in the music as they play this perfection of a map.

_written by [/_DUSK/_](https://osu.ppy.sh/users/6092181)_

[![](/wiki/shared/news/2020-01-05-beatmap-spotlights-autumn-2019/taiko/chalon.jpg)](https://osu.ppy.sh/beatmapsets/999585)

> In the third place we have Good-bye Chalon, an instrumental track by Nekomata Master where [/_DUSK/_](https://osu.ppy.sh/users/6092181) perfectly managed to capture the rustic feeling of the music.

Reviewer 2: ```suggestion
In the third place, we have Good-bye Chalon, an instrumental track by Nekomata Master where [/_DUSK/_](https://osu.ppy.sh/users/6092181) perfectly managed to capture the rustic feeling of the music.
```

Slider Velocity is seamlessly integrated into the map, especially in the variable BPM sections to compliment the frequent intensity changes of the music which gives an incredible amount of depth to the map. The map contains a hefty share of technical and stream sections which leaves no room for the player to rest for the entirety of it. If there’s one person who knows how to map a speedy Camellia song properly, it’s this guy.

_written by [Kazu](https://osu.ppy.sh/users/920861)_

## <a name="catch" id="catch"></a>osu!catch

[![](/wiki/shared/news/2020-01-05-beatmap-spotlights-autumn-2019/catch/cold.jpg)](https://osu.ppy.sh/beatmapsets/946446)

Many mappers may assume that it's hard to be creative on calm and slow songs, but [-Ken](https://osu.ppy.sh/users/4430811) proves that slow maps can also be mapped with creativity. Though it's -Ken's first ranked map, the map itself is really clean and full of amazing patterns. 

> Especially for the kiai parts, the mapper uses a lot of high velocity sliders, which is not common in slow maps, but these sliders fits the map pretty well and gives players a pretty deep impact on this unique mapping style. The flows are carefully arranged and good for all players. Even with the hyper sliders, it's not hard for new players to catch. Spare some time and casually enjoy this masterpiece!

Reviewer 2: ```suggestion
Especially for the kiai parts, the mapper uses a lot of high velocity sliders, which is not common in slow maps, but these sliders fit the map pretty well and give the player a pretty deep impact on this unique mapping style. The flows are carefully arranged and good for all players. Even with the hyper sliders, it's not hard for new players to catch. Spare some time and casually enjoy this masterpiece!
```

But do not forget the guest mappers either, [Daletto](https://osu.ppy.sh/users/7592136), [Greaper](https://osu.ppy.sh/users/2369776), [Nuvolina](https://osu.ppy.sh/users/10974170) and [Sanyi](https://osu.ppy.sh/users/7496029) also contributed into this set, Each of these guys approached the Gothic Hardtrapcore genre differently so be sure to take a look at their part too. Who knows, you might fall in love with this set just as much as I did!

_written by [Kurokami](https://osu.ppy.sh/users/260933)_

[![](/wiki/shared/news/2020-01-05-beatmap-spotlights-autumn-2019/catch/hype.jpg)](https://osu.ppy.sh/beatmapsets/1009824)

Team osu!catch is here to take a place in this season's Spotlights with not one, nor two, but thirteen mappers!

> Dreamland, which is the top difficulty, is mapped by [Ascendance](https://osu.ppy.sh/users/2931883) and [Spectator](https://osu.ppy.sh/users/702598) in collab. Everyone is surely already familiar with them and probably know that they are here for a rare treat. Without even looking at who mapped which part it is easy to recognize their style although they are blending into each other in a godly manner. From the calm mid-section through the build-ups and instense intro to the kiais, this map is perfectly orchestrated and delivers what you expect from both of the mappers.

Reviewer 2: ```suggestion
Dreamland, which is the top difficulty, is mapped by [Ascendance](https://osu.ppy.sh/users/2931883) and [Spectator](https://osu.ppy.sh/users/702598) in collab. Everyone is surely already familiar with them and probably know that they are here for a rare treat. Without even looking at who mapped which part it is easy to recognize their style although they are blending into each other in a godly manner. From the calm mid-section through the build-ups and intense intro to the kiais, this map is perfectly orchestrated and delivers what you expect from both of the mappers.
```

> Next up we have [Minato Yukina](https://osu.ppy.sh/users/531253), [Rocma](https://osu.ppy.sh/users/566276) and a collab between [Ascendance](https://osu.ppy.sh/users/2931883) and [alienflybot](https://osu.ppy.sh/users/636114). They each offer different approach on the same difficulty level which are on par, if not better, with the top difficulty. Minato Yukina decided to follow the melody which resulted a streamy map with intense hyperjump sections, Rocma stands in the middle in terms of rhythm but he has wild patterns. Lastly, do not be fooled with the lower star rating for the collab, this is a real beast on its own, be prepared.

Reviewer 2: ```suggestion
Next up we have [Minato Yukina](https://osu.ppy.sh/users/531253), [Rocma](https://osu.ppy.sh/users/566276) and a collab between [Ascendance](https://osu.ppy.sh/users/2931883) and [alienflybot](https://osu.ppy.sh/users/636114). They each offer a different approach on the same difficulty level which is on par, if not better, with the top difficulty. Minato Yukina decided to follow the melody which resulted in a streamy map with intense hyperjump sections, Rocma stands in the middle in terms of rhythm but he has wild patterns. Lastly, do not be fooled with the lower star rating for the collab, this is a real beast on its own, be prepared.
```

Going further down the list comes [JBHyperion](https://osu.ppy.sh/users/4879508), [Benita](https://osu.ppy.sh/users/4023183) and yet another collab but now between [Sorcerer](https://osu.ppy.sh/users/1913190) and [Bunnrei](https://osu.ppy.sh/users/829284). These guys took on the challenge and tried to approach the Rain difficulty differently, from low circle size to mostly 1/4 patterns. Do not be afraid if you can't handle one of these, try the others as well!

> But lets not forget the rest of the team either. We still have a lot to speak of. The mapset features two Platter difficulty by [ZiRoX](https://osu.ppy.sh/users/200768) and a collab between [Ascendance](https://osu.ppy.sh/users/2931883) and [Mbomb](https://osu.ppy.sh/users/3071175) each with their own mapping style fitting this difficulty. Then we have [Jemzuu](https://osu.ppy.sh/users/7890134) and [Sanyi](https://osu.ppy.sh/users/7496029) with a Salad and Cup respectively. These guys really went out and made this mapset fitting for everyone.

Reviewer 2: ```suggestion
But let's not forget the rest of the team either. We still have a lot to speak of. The mapset features two Platter difficulty by [ZiRoX](https://osu.ppy.sh/users/200768) and a collab between [Ascendance](https://osu.ppy.sh/users/2931883) and [Mbomb](https://osu.ppy.sh/users/3071175) each with their own mapping style fitting this difficulty. Then we have [Jemzuu](https://osu.ppy.sh/users/7890134) and [Sanyi](https://osu.ppy.sh/users/7496029) with a Salad and Cup respectively. These guys really went out and made this mapset fitting for everyone.
```

But we still have two more names to mention who took part in this, [/-Tochi](https://osu.ppy.sh/users/3664366) created a wonderful storyboard and that is accompanied by wonderful hitsounds by [/-Luminate](https://osu.ppy.sh/users/4778689). Together they created something really awesome. Congratulations guys!

_written by [Kurokami](https://osu.ppy.sh/users/260933)_

[![](/wiki/shared/news/2020-01-05-beatmap-spotlights-autumn-2019/catch/kill.jpg)](https://osu.ppy.sh/beatmapsets/1005623)

> Kill the beat by [JBHyperion](https://osu.ppy.sh/users/4879508) is another exciting electric song that hit the ranked category. The map consistently requires quick movements. In addition, the long sliders during the kiai of each difficulty requires accurate rhythmic jumps, letting players truly catch (to) the beat. If you do like electronic music and are up for a challenge, do give this a try.

Reviewer 2: ```suggestion
Kill the beat by [JBHyperion](https://osu.ppy.sh/users/4879508) is another exciting electric song that hit the ranked category. The map consistently requires quick movements. In addition, the long sliders during the kiai of each difficulty require accurate rhythmic jumps, letting players truly catch (to) the beat. If you do like electronic music and are up for a challenge, do give this a try.
```

_written by [Snowless](https://osu.ppy.sh/users/4316266)_

[![](/wiki/shared/news/2020-01-05-beatmap-spotlights-autumn-2019/catch/high.jpg)](https://osu.ppy.sh/beatmapsets/930513)

It is quite refreshing to have a K-pop in the ranked category. [Jemzuu](https://osu.ppy.sh/users/7890134)’s ranked map features an upbeat Korean song. The map is filled with interesting patterns across multiple difficulties that feel rewarding to play because of appropriately distanced jumps and little bits of antiflow that spices the map up, setting it apart from many others that may feel too standardized. Do please try this mapset, you won’t regret it!

_written by [Snowless](https://osu.ppy.sh/users/4316266)_

[![](/wiki/shared/news/2020-01-05-beatmap-spotlights-autumn-2019/catch/conflict.jpg)](https://osu.ppy.sh/beatmapsets/981762)

[Bunnrei](https://osu.ppy.sh/users/829284) jumps into this season's Spotlights with his mapset on siromaru + cranky's "conflict". This set has no kiai but that doesn't make it easier does it? If anything, it makes more interesting!

> The Overdose features 1/4 jumps in quick succession as well as pattern repetition which does add to the difficulty. If you are looking for something which does not joke around the patterns and keeps them as consistent as possible, look no further, you just found it. Alongside this we have high slider velocity which makes some pattern very interesting and provides yet another nice challenge.

Reviewer 2: ```suggestion
The Overdose features 1/4 jumps in quick succession as well as pattern repetition which does add to the difficulty. If you are looking for something which does not joke around the patterns and keeps them as consistent as possible, look no further, you just found it. Alongside this, we have high slider velocity which makes some patterns very interesting and provides yet another nice challenge.
```

Be sure to tune in even if this is not your prefered game mode, this is a real treat!

_written by [Snowless](https://osu.ppy.sh/users/4316266)_

[![](/wiki/shared/news/2020-01-05-beatmap-spotlights-autumn-2019/catch/calling.jpg)](https://osu.ppy.sh/beatmapsets/649285)

Finally we have the calling map in the ranked section. Calling has been expected for ranked for 2 years and [alienflybot](https://osu.ppy.sh/users/636114) is always pushing it better and better. Despite the amazing aesthetic flows and streams, this map also has the epic hitsounds like other maps by the mapper himself. While enjoying the top difficulty, you may also try out other difficulties since the whole set is well done.

_written by [Yumeno Himiko](https://osu.ppy.sh/users/1806962)_

[![](/wiki/shared/news/2020-01-05-beatmap-spotlights-autumn-2019/catch/getaway.jpg)](https://osu.ppy.sh/beatmapsets/1032859)

> [Crowley](https://osu.ppy.sh/users/6341006) is definitely an underrated catch mapper. He mapped western songs a lot, and each map created an unique feeling. Getaway is no doubt the same. With cs5 and ar8.4 map setting, the map feels pretty casual and comfortable, but much different from other maps. As one of the maps in osu!catch Carpool Tunnel Mini-pack, it is a good featured artist map for middle and top players. Take care of your accuracy as this map has many droplets throughout sliders.

Reviewer 2: ```suggestion
[Crowley](https://osu.ppy.sh/users/6341006) is definitely an underrated catch mapper. He mapped western songs a lot, and each map created a unique feeling. Getaway is no doubt the same. With cs5 and ar8.4 map setting, the map feels pretty casual and comfortable, but much different from others. As one of the maps in osu!catch Carpool Tunnel Mini-pack, it is a good featured artist map for middle and top players. Take care of your accuracy as this map has many droplets throughout sliders.
```

---
layout: post
title: "Beatmap Spotlights: Autumn 2019"
date: 2020-01-05 10:00:00 +0000
---

We have now entered a new astronomical decade and along with that the winds are not chilling but we still have something to warm you up! Our selection from the season of fallen leaves is here. Come, give them a look and earn a fancy new medal for yourself!

> ![](/wiki/shared/news/2020-01-05-beatmap-spotlights-autumn-2019/header.jpg)

Reviewer: ```suggestion
![](/wiki/shared/news/banners/spotlights-header.jpg)
```

Author: I forgot to add the header images.

If you have enough time, be sure to check this one out!

_written by [KnightC0re](https://osu.ppy.sh/users/7894340)_

[![](/wiki/shared/news/2020-01-05-beatmap-spotlights-autumn-2019/osu/last.jpg)](https://osu.ppy.sh/beatmapsets/1015203)

Stop arguing over whether flawless playability and high-end aesthetic can exist in the same map - [captin1](https://osu.ppy.sh/users/689997) and [Kalibe](https://osu.ppy.sh/users/3376777) have proven in their collaboration work, lmy - Last words, that it is completely possible. Multiple takes on emphasis combining with positively weird but well-designed slider patterns bring a whole new playing experience set to Aitsuki Nakuru's beautiful voice.

_written by [Ryuusei Aika](https://osu.ppy.sh/users/7777875)_

[![](/wiki/shared/news/2020-01-05-beatmap-spotlights-autumn-2019/osu/natsu.jpg)](https://osu.ppy.sh/beatmapsets/1006502)

> Coupled with some amazing keysounding, the top difficulty uses numerous equal-distance obtuse note patterns to create a flow with slightly increasing difficulty to accurately represent the little crescendos in the piano melodies, and makes ample use of various distance gimmicks under low approach rate plus clearly distinguishable time-distance gaps to maximize the representation of the song's atmosphere in all of its different parts. 

Reviewer 3: This run on sentence gets really difficult to read. There's way too much going on, the different points should be seperated into sentences where fitting - this would require rewriting.

Author: No idea how to separate this as I have zero clue on the map itself

Author: Hopefully fixed

[![](/wiki/shared/news/2020-01-05-beatmap-spotlights-autumn-2019/taiko/chalon.jpg)](https://osu.ppy.sh/beatmapsets/999585)

In the third place, we have Good-bye Chalon, an instrumental track by Nekomata Master where [\_DUSK\_](https://osu.ppy.sh/users/6092181) perfectly managed to capture the rustic feeling of the music.

As the song itself is fairly complex, the Inner Oni difficulty features a lot of patterns on irregular snaps while maintaining impeccable flow, which results in a very satisfying gameplay experience as it provides players with a nice accuracy challenge while flawlessly following the music. Apart from the patterns themselves, the SV usage does a very good job at representing the intensity of each section which makes this a great map in terms of visual responsiveness.

The Oni difficulty gives intermediate players a chance to shine as it is structured in a way that it doesn't become too complex for the target audience while still being extremely fun to play!

_written by [Genjuro](https://osu.ppy.sh/users/3196091)_

[![](/wiki/shared/news/2020-01-05-beatmap-spotlights-autumn-2019/taiko/betrayal.jpg)](https://osu.ppy.sh/beatmapsets/1026679)

> The obligatory metal beatmap for the Autumn spotlights is Inferi - A Betrayal Unforetold, a Featured Artist song mapped by [Raiden](https://osu.ppy.sh/users/2239480) and [Grünschale/Greenshell](https://osu.ppy.sh/users/Greenshell) (with a storyboard by [Faputa](https://osu.ppy.sh/users/Faputa)!.

Reviewer 3: ```suggestion
The obligatory metal beatmap for the Autumn spotlights is Inferi - A Betrayal Unforetold, a Featured Artist song mapped by [Raiden](https://osu.ppy.sh/users/2239480) and [Grünschale/Greenshell](https://osu.ppy.sh/users/Greenshell) (with a storyboard by [Faputa](https://osu.ppy.sh/users/Faputa)!).
```

In 5th place we have Silentroom x Tanchiky - Outer Solution, which is an approach to MEGALOVANIA from the UNDERTALE series, one of the most popular games this year.

About the mapping, there's just so much to say about it that we could keep talking about it for hours: Even though [rubies87](https://osu.ppy.sh/users/4949934) is a rather new mapper with this being his first apparition in the ranked section, it truly shines as one of the best creations that not even the most experienced mappers can possibly do.

With an incredible judgment of the tools the song provided him to make an excellent approach that could at first look incoherent to the song but ends up adding a layer to it that makes it much more rich and an adequate usage of SV that is perfectly readable on a first try, makes this map not only spotlight worthy but also a good reason to believe how promising rubies87 is as a mapper.

_written by [\_DUSK\_](https://osu.ppy.sh/users/6092181)_

[![](/wiki/shared/news/2020-01-05-beatmap-spotlights-autumn-2019/taiko/psystyle.jpg)](https://osu.ppy.sh/beatmapsets/1022545)

> Closing this seasons (and year) spotlights we have Camellia - We Could Get More Machinegun Psystyle! (And More Genre Switches) mapped by [Alchyr](https://osu.ppy.sh/users/4993032), a popular individual who has created by far one of the most unique speed maps of the year. 

Reviewer 3: ```suggestion
Closing this season's (and year's) spotlights we have Camellia - We Could Get More Machinegun Psystyle! (And More Genre Switches) mapped by [Alchyr](https://osu.ppy.sh/users/4993032), a popular individual who has created by far one of the most unique speed maps of the year. 
```

But do not forget the guest mappers either, [Daletto](https://osu.ppy.sh/users/7592136), [Greaper](https://osu.ppy.sh/users/2369776), [Nuvolina](https://osu.ppy.sh/users/10974170) and [Sanyi](https://osu.ppy.sh/users/7496029) also contributed into this set, Each of these guys approached the Gothic Hardtrapcore genre differently so be sure to take a look at their part too. Who knows, you might fall in love with this set just as much as I did!

_written by [Kurokami](https://osu.ppy.sh/users/260933)_

[![](/wiki/shared/news/2020-01-05-beatmap-spotlights-autumn-2019/catch/hype.jpg)](https://osu.ppy.sh/beatmapsets/1009824)

Team osu!catch is here to take a place in this season's Spotlights with not one, nor two, but thirteen mappers!

> Dreamland, which is the top difficulty, is mapped by [Ascendance](https://osu.ppy.sh/users/2931883) and [Spectator](https://osu.ppy.sh/users/702598) in collab. Everyone is surely already familiar with them and probably know that they are here for a rare treat. Without even looking at who mapped which part it is easy to recognize their style although they are blending into each other in a godly manner. From the calm mid-section through the build-ups and intense intro to the kiais, this map is perfectly orchestrated and delivers what you expect from both of the mappers.

Reviewer 3: ```suggestion
Dreamland, which is the top difficulty, is mapped by [Ascendance](https://osu.ppy.sh/users/2931883) and [Spectator](https://osu.ppy.sh/users/702598) in collab. Everyone is surely already familiar with them and probably know that they are here for a rare treat. Without even looking at who mapped which part it is easy to recognize their styles even when they are blending into each other in a godly manner. From the calm mid-section through the build-ups and intense intro to the kiais, this map is perfectly orchestrated and delivers what you expect from both of the mappers.
```

Despite the contrast between sections, -mint- does an impressive job at keeping coherent concepts consistent through the entire map, such as chord allocation and pitch relevancy. Her emphasis on using long note lengths to represent instrument type and intensity as well as her ability to differentiate regular and long notes depending on the instruments used and the intensity of the song is very impressive and shows her prowess in making long note maps. Overall the map is an absolute joy to play and the fantastic concepts used make it more than worthy of a place in Spotlights!

_written by [\-MysticEyes](https://osu.ppy.sh/users/6253266)_

[![](/wiki/shared/news/2020-01-05-beatmap-spotlights-autumn-2019/mania/education.jpg)](https://osu.ppy.sh/beatmapsets/983953)

[Kamuy](https://osu.ppy.sh/users/7439226) successfully managed to capture the harsh and repetitive nature of PSYQUI's Education and has transformed it into a tough map that is not for the faint of heart! The chart cleverly exploits minijacks and short LNs to artificially induce the feeling of roughness and make the map harder to nail off, especially for players with ailing endurance.

> Kamuy's implementation of the map's repetition also does justice to the song without actually making the map feel repetitive, using variations of structuring throughout the entire map. In general, the map is a blast to play to and as a medium to enjoy the song. If you are not up for the challenge though, ByeForNow's difficulty and all of the easier difficulties still follow the same ideologies while sparing you a fight.

Reviewer 4: ```suggestion
Kamuy's implementation of the map's repetition also does justice to the song without actually making the map feel repetitive, using variations of structuring throughout the entire map. In general, the map is a blast to play to and as a medium to enjoy the song. If you are not up for the challenge though, [ByeForNow](https://osu.ppy.sh/users/71991590)'s difficulty and all of the easier difficulties still follow the same ideologies while sparing you a fight.
```

_written by [Toaph Daddy](https://osu.ppy.sh/users/7616811)_

---

We've started to migrate over to our dedicated website properly in recent months, as our existing infrastructure was becoming increasingly tedious and outdated. In the near future, we'll have much more of what we do visible over at our brand new site, but for now, only the map selection is available. If you're interested, check it out [here](https://spotlights.team/spotlights-results).

We hope you enjoy our selection for this season and look forward to what we will bring to you in the future. Until then, happy circle clicking, drum bashing, fruit catching, and key smashing! Do not forget to have a rest occasionally!

> —the Spotlights Team

Reviewer 3: shouldn't this be signed Kurokami as the author of the post like previous spotlights.

Author: I am still not a fan of that, this is a team effort and not mine alone

Reviewer 5: this part is just telling who wrote the uncredited parts of the post, not who contributed to the whole project

Author: I do not see it that way and i rather not want to be credited for literal copy pasting with some extra

Reviewer 5: when we went back and changed all of the other authors, I think it was because the beginnings and ends of these posts are left with a false author---the whole team didn't write it, you did

in all fairness it's not really wrong to say "by the spotlights team" concerning the whole post cuz most of the team did write something. just feels weird to me cuz u already give credit to everyone else and the author doesn't mean anything more than who wrote the beginning/end

Author: Just because I add something to the beginning and at the end that doesn't make me author in my opinion. Look at it this way, everyone writes a tiny piece which means that it is done by the whole team, not just one person. Just because we write their names under each part that does not change this. And I was against the change in the previous Spotlights articles as well because of the same reason.

Reviewer 5: I'm the person who started authoring posts "by the Project Loved team", I have looked at it this way before and that's what I'm saying in my second paragraph of last message. it turned out to be a bad idea because it falsely credited the whole team for the whole post, and nobody could tell who wrote the actual announcement beyond of each map description unless they checked github (we got questions about this for Loved which was why I looked into changing it back to names in the first place)

none of this is a big deal at all, but there are clear minor benefits to using your name as the author so I don't see why you shouldn't

Author: I do not mind if they do not know exactly who writes the announcement. I do not want to take credit for like 1% of the job when it mostly only consists of copy-pasting.

Reviewer 5: I know you don't mind about that, I didn't either, but apparently some readers do. idk why you think of this as "taking credit" when all spotlights guys are already credited for exactly what they wrote. in the time we've been authoring team posts like this I've seen no complaints or confusion

Author: I never saw anyone complaining about this on Spots articles, they rather point out the small number of maps. 👀 The beginning and the end is so insignificant that they might just not even read it. I barely write anything and soon, thanks to Venix, even less.

Reviewer 5: well I can't make you put ur name down if you don't want to, I think this is a strange decision considering how every other 'team' post is written
